### ISSUE TITLE
a question about DCGAN

### ISSUE THREAD

**Issue by XiaoHao-Chen** [HelenMao/MSGAN#9]
Thank you very much for your excellent work!
Author Hello, I would like to ask, when you test is based on what indicators to select the load training saved model? What are the indicators that let you know that the generator is perfect at this time and can be used for testing to  generate image? Because in training, only the number of iterations and the learning rate are printed in real time.
Another question is, can the program you provide run on more GPUs?
Thank you very much for your help!

**Comment by HelenMao** [comment]
Hi, thanks for your attention.
In our experiment, we just monitored the quality of the generated images in tensorboardX. Then, we calculated the FID offline and choose the best model. 
One GPU is enough for the training process of DCGAN, since the model is very simple.

**Comment by lyt1001** [comment]
Hello，Thanks for sharing the great work! 
I would like to ask some questions since I think DRIT is kind of hard for me.
Num1: What does fake_B_encoded、fake_B_random and fake_AA_encoded respectively mean? In other words, I don't konw the mean of output_fakeA of generators. (model.py line 153)
Num2: I have trained my DRIT model using cat2dog dataset under default train options for 1000 epochs , I think maybe it's time to test. So I test my model by "--dataroot ./datasets/cat2dog --resume ./results/trial/00979.pth"and get 5 dogs images per folder. But the 5 dogs looks all same, the only difference is color. Does that mean mode collapse occur or I test my model in a wrong way? How can I get the results like your "Example Results"？
Num3: How can I calculated scores such as FID or LPIPS?
Thanks again!
